src/components/Editor/SearchBar.js

```diff
@@ -65,7 +67,7 @@ const SearchBar = React.createClass({
     }
   },
 
-  componentDidUpdate(prevProps) {
+  componentDidUpdate(prevProps: any) {
```

Comment:
You can use `Props` here. https://github.com/facebook/flow/blob/v0.35.0/lib/react.js#L38